web/html/src/components/table/TableDataHandler.tsx

```diff
@@ -375,40 +378,45 @@ export class TableDataHandler extends React.Component<Props, State> {
 
     const emptyText = this.props.emptyText || t("There are no entries to show.");
     const isSelectable = typeof this.props.selectable !== "undefined" && this.props.selectable !== false;
-
+    const isHeader = this.props.hideHeaderFooter === "header" || this.props.hideHeaderFooter === "both";
```

Comment:
This name is somewhat ambiguous, would recommend either `hideHeader` or `hasHeader` depending on whether you want to make the check inverse or not.